okapi/core.php

```diff
@@ -28,7 +28,7 @@
 function get_admin_emails()
 {
     $emails = array();
-    if (class_exists("okapi\\Settings"))
+    if (class_exists(Settings::class))
```

Comment:
This change doesn't make much sense, I think. The purpose of this line seems to be to check if the class exists. Previously, if this class didn't exist, `class_exists` would return `false`. After your change, I think that an exception will be thrown, even before `class_exists` is called.